lorawan-encoding/src/maccommandcreator.rs

```diff
@@ -314,6 +314,17 @@ impl RXParamSetupAnsCreator {
 
         self
     }
+
+    /// Get acknowledgment value
+    pub fn ack_get(&self) -> u8 {
+        self.data[1]
+    }
+
+    /// Set acknowledgment value
+    pub fn ack_set(&mut self, ack: u8) -> &mut Self {
```

Comment:
I am not a fan of  letting the user of this API inject any u8 in to here as all values of u8 are not valid.

Reply:
Yeah, I just needed something fast. Anyway, I think I can actually get rid of this API once I rework the uplink cleanup using same approach as you used.

Reply:
just save the whole RXParamSetupAnsCreator then. It is only one byte extra. I don't like putting "temporary" stuff in crates that are used outside of lora-rs.

Reply:
I reworked the acknowledgment handling by retaining the mac commands and this allowed me to get rid of this API.